backend/src/common/server-application/base-server-app-api.ts

```diff
@@ -1,7 +1,7 @@
+import { AppEnvironment } from 'shared';
```

Comment:
same here.
I also want to ask that in the future, after you've received a review, please check the changes you've made to ensure that similar issues are avoided. We may not point out every instance.